### PR TITLE
Multiple busybox app example and to add MDR missing readme

### PR DESCRIPTION
**Pull request by Akarsha-rai**
- Examples to deploy multiple busybox applications 
- To add MDR missing readme as per the issue #35

### CODE REVIEW

File: mdr/subscriptions/busybox-app-1/subscriptions/busybox/drpc.yaml [hunk 1]
```diff
@@ -6,12 +6,12 @@ metadata:
   labels:
     app: busybox-sample
 spec:
-  preferredCluster: "akrai-s19-c1"
```
Comment (prsurve) [inline mdr/subscriptions/busybox-app-1/subscriptions/busybox/drpc.yaml:9]: We should not hardcode cluster name here

File: mdr/subscriptions/busybox-app-3/subscriptions/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+namespace: rramen-busybox-3
```
Comment (prsurve) [inline mdr/subscriptions/busybox-app-3/subscriptions/kustomization.yaml:1]: typo here

File: mdr/subscriptions/busybox-app-1/subscriptions/busybox/drpc.yaml (hunk 1 above)
Comment (Akarsha-rai) [reply mdr/subscriptions/busybox-app-1/subscriptions/busybox/drpc.yaml:9]: done

File: mdr/subscriptions/busybox-app-3/subscriptions/kustomization.yaml (hunk 2 above)
Comment (Akarsha-rai) [reply mdr/subscriptions/busybox-app-3/subscriptions/kustomization.yaml]: updated

File: mdr/subscriptions/busybox-app-1/subscriptions/busybox/drpc.yaml (hunk 1 above)
Comment (Akarsha-rai) [reply mdr/subscriptions/busybox-app-1/subscriptions/busybox/drpc.yaml:9]: done

**Review by prsurve — APPROVED** [review]

**Review by parikshithb — APPROVED** [review]

File: mdr/subscriptions/busybox-app-1/resources/deployment/busybox-cephfs-pod.yaml [hunk 3]
```diff
@@ -19,10 +18,11 @@ spec:
...
       - command: ["/bin/sh", "-c"]
         args:
-          - wget  https://raw.githubusercontent.com/red-hat-storage/ocs-workloads/io-pattern-busybox/rdr/busybox/scripts/simple_io.sh -O /mnt/test/simple_io.sh;
-            chmod +x /mnt/test/simple_io.sh;
+          - wget  https://raw.githubusercontent.com/red-hat-storage/ocs-workloads/master/rdr/busybox/scripts/simple_io.sh -O /mnt/test/simple_io.sh;
+            wget https://raw.githubusercontent.com/red-hat-storage/ocs-workloads/master/rdr/busybox/scripts/simple_io_integrity_check.sh  -O /mnt/test/simple_io_integrity_check.sh;
+            chmod +x /mnt/test/simple_io.sh; chmod +x /mnt/test/simple_io_integrity_check.sh;
             /mnt/test/simple_io.sh
-        image: quay.io/vcppds7878/busybox:latest
+        image: quay.io/ocsci/rdr-ocs-workload
```
[the remark this answers is not in the document]
Comment (prsurve) [reply mdr/subscriptions/busybox-app-1/resources/deployment/busybox-cephfs-pod.yaml:25]: yes you are right

[the remark this answers is not in the document]
Comment (prsurve) [reply mdr/subscriptions/busybox-app-1/resources/deployment/busybox-cephfs-pod.yaml:25]: https://github.com/red-hat-storage/ocs-workloads/blob/master/rdr/busybox/app-busybox-1/resources/deployment/busybox-dc-1.yaml#L26 It wont matter be it busybox or rdr-ocs-workload as both the image base is busybox

[the remark this answers is not in the document]
Comment (prsurve) [reply mdr/subscriptions/busybox-app-1/resources/deployment/busybox-cephfs-pod.yaml:25]: And to run above workload we need rdr-ocs-workload  image

**Pull request merged**